### DISCUSSION TITLE
Average electrolyte concentration not constant

### DISCUSSION THREAD

**Discussion by AC-J** [pybamm-team/PyBaMM#3464] [Q&A] [answered]
Hi together,
I am currently running some simulations with a DFN-Half-Cell Model (Pybamm Version 23.3). I noticed that the average electrolyte concentration does drop significantly, if the electrolyte diffusivity is low (e.g. 1e-13 m2.s-1). The parameter set I intend to use has an even lower coefficient. I have over 25% loss, which increases with higher C-rate. I have tried increasing the resolution and decreasing the tolarances, but that has not helped.
As far as I know, the average electrolyte concentration should stay constant, right? Maybe someone has an idea why this happens?

Example:
```
import pybamm
import numpy as np

model = pybamm.lithium_ion.DFN(options={'working electrode': 'positive'}) # more or less the same occurs with pybamm.lithium_ion.BasicDFNHalfCell
param = model.default_parameter_values
param.update({
    'Electrolyte diffusivity [m2.s-1]': 1e-13
    }, check_already_exists=False)
sim = pybamm.Simulation(model, C_rate=0.2, parameter_values=param)
sim.solve(np.linspace(0, 3600/0.2, 501))

c_e = sim.solution['X-averaged electrolyte concentration [mol.m-3]'].entries
```

Best regards

**Comment by brosaplanella** [comment, marked as answer]
What should stay constant is the total lithium in electrolyte, not just the concentration. In this case, what you want to consider is the concentration times the porosity, which is the quantity that should remain constant. The variable `"Total lithium in electrolyte [mol]"` captures this quantity.

**Reply by AC-J** [reply]
Thank you very much. I've get it now.
